sklearn/utils/estimator_checks.py

```diff
@@ -131,6 +131,30 @@ def _yield_classifier_checks(name, Classifier):
     if 'class_weight' in Classifier().get_params().keys():
         yield check_class_weight_classifiers
 
+def check_supervised_y_no_nan(name, Estimator):
+    # Checks that the Estimator targets are not NaN.
+
+    warnings.simplefilter("ignore")
+    np.random.seed(888)
```

Comment:
nit: Can you replace with this rng = np.random.RandomState(888)

just to be consistent with others.